go.mod

```diff
@@ -122,3 +124,5 @@ require (
 )
 
 go 1.12
+
+replace github.com/ipfs/go-ipfs-config => github.com/marten-seemann/go-ipfs-config v0.2.5-0.20190418001109-c95be0eacc9f
```

Comment:
@marten-seemann FYI, you can test PRs that depend on other PRs by adding replace statements.

Reply:
I'm aware of that, but I didn't want to put it in a PR, because we might merge it accidentally without removing the `replace` first.

Reply:
I wonder if there's a bot we could employ for that.